sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryExecutionErrors.scala

```diff
@@ -362,12 +362,16 @@ private[sql] object QueryExecutionErrors extends QueryErrorsBase with ExecutionE
         "groupIndex" -> groupIndex.toString()))
   }
 
-  def invalidUrlError(url: UTF8String, e: URISyntaxException): SparkIllegalArgumentException = {
+  def invalidUrlError(
+      url: UTF8String,
+      e: URISyntaxException,
+      suggestedFunc: String)
+  : SparkIllegalArgumentException = {
     new SparkIllegalArgumentException(
       errorClass = "INVALID_URL",
       messageParameters = Map(
         "url" -> url.toString,
-        "ansiConfig" -> toSQLConf(SQLConf.ANSI_ENABLED.key)),
+        "functionName" -> suggestedFunc),
```

Comment:
Could we please just push try_parse_url to the error-conditions and not have this parameter. If the need comes that we have different functionNames we will update, but for now we want to push as much of the text as possible to error-conditions